scripts/ci/install.sh

```diff
@@ -19,26 +19,58 @@
 set -e
 set -u
 
-if [ -z "$TRAVIS_OS_NAME" ]; then
-  echo "This script is only meant to run in Travis CI" 1>&2
+function usage() {
+  echo "Usage: $0"
+  echo ""
+  echo "Options"
+  echo ""
+  echo "    -o <operating system>"
+  echo "    -r <architecture>"
   exit 1
+}
+
+ARGV_OPERATING_SYSTEM=""
+ARGV_ARCHITECTURE=""
+
+while getopts ":o:r:" option; do
+  case $option in
+    o) ARGV_OPERATING_SYSTEM=$OPTARG ;;
+    r) ARGV_ARCHITECTURE=$OPTARG ;;
+    *) usage ;;
+  esac
+done
+
+if [ -z "$ARGV_OPERATING_SYSTEM" ] || [ -z "$ARGV_ARCHITECTURE" ]; then
+  usage
 fi
 
-if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
-    -r "$TARGET_ARCH" \
+    -r "$ARGV_ARCHITECTURE" \
     -s "$(pwd)" \
     -c "make info && make electron-develop"
 else
   ./scripts/build/check-dependency.sh pip
-  ./scripts/build/check-dependency.sh brew
   ./scripts/build/check-dependency.sh make
   ./scripts/build/check-dependency.sh npm
```

Comment:
Personally, I'd re-order these lines to match the order the commands are used in, i.e. `npm, pip, make`. But that's obviously a very minor thing and doesn't really matter.